lib/utils/getimage.dart

```diff
@@ -0,0 +1,6 @@
+import 'package:pokedex/utils/strings.dart' as str;
+import 'package:pokedex/utils/constant.dart' as k;
+
+extension GetImage on String {
```

Comment:
kindly rename this to ImageUrl or whatever is best suited 🤔 

also this dart file is supposed to be for extension purposes right? kinda confused why it's named `getImage.dart`? 🤔 rename this instead to `extensions.dart`